src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -750,13 +750,13 @@ public Page<VisitWithVisitors> getBookingVisitsWithVisitor(
         @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.", content = {@Content(mediaType = "application/json", schema = @Schema(implementation = ErrorResponse.class))})})
     @Operation(summary = "Balances visit orders and privilege visit orders for offender.", description = "Balances visit orders and privilege visit orders for offender.")
     @GetMapping("/offenderNo/{offenderNo}/visit/balances")
-    @VerifyOffenderAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VISIT_SCHEDULER"})
+    @VerifyOffenderAccess(overrideRoles = {"GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VISIT_SCHEDULER"})
     @SlowReportQuery
     public VisitBalances getBookingVisitBalances(
         @PathVariable("offenderNo") @Parameter(description = "The offenderNo of offender", required = true) final String offenderNo,
         @RequestParam(value = "allowNoContent", required = false) @Parameter(description = "Allow no content (204) response if no data rather than returning a not found (404)") final boolean allowNoContent
     ) {
-        final var identifiers = bookingService.getOffenderIdentifiers(offenderNo, false, "SYSTEM_USER", "VISIT_SCHEDULER").getBookingAndSeq().orElseThrow(EntityNotFoundException.withMessage("No bookings found for offender {}", offenderNo));
+        final var identifiers = bookingService.getOffenderIdentifiers(offenderNo, false, "GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VISIT_SCHEDULER").getBookingAndSeq().orElseThrow(EntityNotFoundException.withMessage("No bookings found for offender {}", offenderNo));
```

Comment:
We now have duplicate the overrideRoles values  on the endpoint (this was probably incorrect anyway) when we getOffenderIdentifers (which in turn does a verifyOffender/BookingAccess).
This is because all clients calling this endpoint with SYSTEM_USER were allowed to access the data, and still need to be.  The tests show that the alternative roles are now successful.

Reply:
`VerifyOffenderAccess` also has `VISIT_SCHEDULER` - how come that's not mentioned on this line too?

Reply:
Also:
> all clients calling this endpoint with SYSTEM_USER were allowed to access the data, and still need to be
So why remove SYSTEM_USER?

Reply:
VISIT_SCHEDULER was already on the getOffenderIdentifers call - so I didn't mention it - it is on
`@VerifyOffenderAccess(overrideRoles = {"GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VISIT_SCHEDULER"})
` matching on
`bookingService.getOffenderIdentifiers(offenderNo, false, "GLOBAL_SEARCH", "VIEW_PRISONER_DATA", "VISIT_SCHEDULER")`

Remove SYSTEM_USER - because we've been told to ?